### PR TITLE
sbt 1.0.0-M4 (devel)

### PR DESCRIPTION
**Pull request by Jerzerak**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---
- Add sbt devel version 1.0.0-M4

### CODE REVIEW

File: Formula/sbt.rb [hunk 1]
```diff
@@ -4,6 +4,12 @@ class Sbt < Formula
   url "https://dl.bintray.com/sbt/native-packages/sbt/0.13.11/sbt-0.13.11.tgz"
   sha256 "a36a6fbf6dd70afd93fb8db16c40e8ac00798fdddfa0b4c678786dc15617afa6"
 
+  devel do
+    url "https://dl.bintray.com/sbt/native-packages/sbt/1.0.0-M4/sbt-1.0.0-M4.tgz"
+    sha256 "8cb2eaabcbfeceeb65023311b08c980feff80552b22524213c71857ced2f8de7"
+    version "1.0.0-M4"
```
Comment (ilovezfs) [inline Formula/sbt.rb]: Interesting. If I remove this, brew thinks the version is "4." That looks like a bug to me. Do you have any idea what the meaning of "M" is here?

Comment (Jerzerak) [inline Formula/sbt.rb]: @ilovezfs not a clue (strange convention if you ask me)... I could ask one of the maintainers if you're really curious

Comment (ilovezfs) [inline Formula/sbt.rb]: Most likely means milestone, if I had to guess. We should probably consider updating the version class to be able to parse this as 1.0.0-M4 automatically.

Comment (Jerzerak) [inline Formula/sbt.rb]: Yeah I have noticed a lot of the devel versions that I've updated recently are being parsed incorrectly...

**Comment by ilovezfs** [comment]
Thank you once again, @Jerzerak :cake:

**Pull request closed without merging**